cloud/disk_manager/deploy/helm/disk-manager/templates/deployment.yaml

```diff
@@ -0,0 +1,54 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata: {{ $fullname := printf "%s-%s" "disk-manager" .Values.serviceKind }}
+  name: {{ $fullname }}
+  namespace: {{ .Values.namespace | default "disk-manager" }}
+  labels:
+    app: {{ $fullname }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version }}
```

Comment:
if you run `helm init mychart` it will generate help char stub inside `_helpers.tpl` you will find helper template 
```
{{/*
Common labels
*/}}
{{- define "mychart.labels" -}}
helm.sh/chart: {{ include "mychart.chart" . }}
{{ include "mychart.selectorLabels" . }}
{{- if .Chart.AppVersion }}
app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
{{- end }}
app.kubernetes.io/managed-by: {{ .Release.Service }}
{{- end }}
```
you should probably define similar template for us and use it to add helm common labels